lti_emailer/settings/base.py

```diff
@@ -307,3 +307,5 @@
 
 CACHE_KEY_MESSAGE_HANDLED_BY_MESSAGE_ID_AND_RECIPIENT = "lti_emailer:message-handled:%s:%s"
 CACHE_KEY_MESSAGE_HANDLED_TIMEOUT = 60 * 60 * 8  # 8 hours
+
+NO_REPLY_ADDRESS = SECURE_SETTINGS.get('no_reply_address', 'no-reply@coursemail.harvard.edu')
```

Comment:
Hrm...surprisingly, they're willing to let us set the from field to a different domain than the sender field...I would have expected them to be too worried about sending reputation to allow that.

https://documentation.mailgun.com/faqs.html#what-is-the-difference-between-the-from-and-sender

Still, if those domains don't match, there's an increased likelihood that email will end up in the spam bucket.